### ISSUE TITLE
Review "bearer of some rare" axioms

### ISSUE THREAD

**Issue by sabrinatoro** [monarch-initiative/mondo#6448]
the "bearer of some rare" axioms were originally added automatically for all the Orphanet rare diseases, and some are manually curated. 

In the context of "rare subset":
- should we keep these axioms coming from Orphanet (in addition to the "orphanet-rare subset")?
- make sure that all the diseases in the rare subset also have a "rare" axiom?

**Comment by sabrinatoro** [comment]
@matentzn can we discuss this in one of our 1:1?

**Comment by sabrinatoro** [comment]
this issue came from this PR: https://github.com/monarch-initiative/mondo/pull/6403

**Comment by sabrinatoro** [comment]
From curation call 230720
- **need a systematic approach to adding the "bearer of some rare"**
- orphanet has "part of" classification, and therefore these axioms might have been added incorrectly

**Comment by sabrinatoro** [comment]
Example: "cat scratch disorder" MONDO:0005692

**Comment by matentzn** [comment]
In my last call with @cmungall we have decided that we should maintain an automatic / dynamic definition of "has bearer some rare". We should 

- throw them out from Mondo (asserted)
- Auto-generate them as a component based on the rare disease authorities we have
- get rid of the `mondo_rare` subset altogether.

**Comment by sabrinatoro** [comment]
@matentzn : 
1. yes, there are 2 dos-dp patterns with "has characteristic some rare": 
    1. https://github.com/monarch-initiative/mondo/blob/master/src/patterns/dosdp-patterns/rare.yaml
    2. https://github.com/monarch-initiative/mondo/blob/master/src/patterns/dosdp-patterns/rare_genetic.yaml
1. I cannot find any example of any diseases with "has bearer of some rare" (or has characteristic of some rare). It looks like this axiom is only used as a SubClassOf annotation (ie no logical definition)

**Comment by matentzn** [comment]
Why do these patterns exist if they are not manifested anywhere?

**Comment by sabrinatoro** [comment]
My guess is that it was for the "rare something disease" terms that have been obsoleted since then. But that's only a guess

**Comment by sabrinatoro** [comment]
@matentzn I created a query (https://api.triplydb.com/s/-Be2kOAPw) to get the terms with 'bearer_of some rare' + their source. (personal celebration on the success of this sparql query creation! 🥳 )

The interesting point is that most of the sources for these axioms are 
- GARD 
- MONDO id for "rare" parent that has been obsoleted (because it was a grouping class term)

There are 13 terms for which the axiom has a source which is a PMID/orcid/website. 
I will add the "mondo_subset" + their sources for these terms.  (Will link the PR here ). After this is done, I think it is safe to throw out all these axioms. I'll let you know when this is done.

**Comment by matentzn** [comment]
Impressive!

**Comment by matentzn** [comment]
One thing that we have not specified about the rare disease authority subsets (`gard_rare`, `orphanet_rare`, `nord_rare`) is when to attach them to a term, if the terms become deprecated. I put my suggestions in brackets. @nicolevasilevsky @sabrinatoro let me know what you think.

- [ ] Decide what happens in the following three scenarios:
        1. MONDO ID is obsolete, external ID is not `[Do NOT include in subset]`. The alternative here would be to `Include them in subset`, but to `exclude them from the overarching `rare` subset? I am uncertain about this one.
        2. MONDO ID is obsolete and external ID is obsolete `[Do NOT include in subset]`
        3. MONDO ID is active, and external ID is obsolete `[Do NOT include in subset]` (Note that the MONDO ID my end up being included in the general `rare` subset, but for another reason)
- [ ] Document the above decision in the rare subset docs

**Comment by sabrinatoro** [comment]
@matentzn I agree with your suggestions: 
- if a term is obsoleted in Mondo, it should be removed from the rare subset (note that we should not obsolete a term that is in a rare subset unless we have very good reasons). 
- if a term is obsoleted in Mondo because it is merged, the term to which it is merged should be in the rare subset
- if a term is obsoleted at the source, it should lose its "rare subset" status

**Comment by matentzn** [comment]
> if a term is obsoleted in Mondo because it is merged, the term to which it is merged should be in the rare subset

This is only true when you move the xrefs to the active term during the merge. 

Can you document these decisions in the rare subset docs, with a short reason (half a sentence)?

**Comment by nicolevasilevsky** [comment]
sounds good to me

**Comment by sabrinatoro** [comment]
Everything in this issue has been addressed and documentation has been created. I am closing this issue

**Comment by matentzn** [comment]
There are two action items remaining, but maybe they have different issues:

- [ ] Remove all bearer of some rare axioms that currently exist in Mondo edit
- [ ] Add pipeline for adding them back automatically from the subsets.. IF (and only IF) needed.

**Comment by sabrinatoro** [comment]
@matentzn Did we make a decision on what happens to the `'has characteristic' some rare` axioms? 
I see some in the ontology that came from "rare ~" disease terms (terms, which have been obsoleted). 
If we haven't discussed this, I will add it on the list of things to discussed related to rare disease.

**Comment by matentzn** [comment]
Not really, no. Its difficult to come up with arguments one way or another - there is no clear use case for keeping them, but there is also no clear use case for removing them. So from a management perspective you need to weight:

1. The fact that you _change_ something, i.e. removing the bearer some rare axioms (which may trip up some users) against
2. Keeping some axioms that do not have a clear purpose in Mondo.

Note: the "rare" characteristic is not used in a logical definition. 
Note2: It seems "rare" has a minor purpose in QC:

<img width="532" alt="image" src="https://github.com/monarch-initiative/mondo/assets/7070631/4ad9cce9-f040-4a79-9891-87626543c73f">

basically there are two diseases which are defined as "not rare" (common cold and hemochromatosis type 1), and there is this GC (see screenshot) that ensures that no disease is rare and "not rare" at the same time.. Not sure how useful this is.

**Comment by sabrinatoro** [comment]
Thank you @matentzn.
I think the current axioms are related to https://github.com/monarch-initiative/mondo/issues/7235. 
IMO, keeping these axioms or not will depend on whether it is useful to users. I think we should either 
(1) remove all these axioms, or 
(2) add these axioms automatically based on terms being in the "rare subset"

 Let's discuss this on a call.

**Comment by sabrinatoro** [comment]
Discussed on the curation call on Feb 16, 2024:
We think that the `"has characteristic" some rare` axioms are unnecessary since we now have the "rare disease" subsets.
We should remove all the `"has characteristic" some rare` axioms 

Note that we could add these axioms at a later time, but
- this should be done ONLY if users ask for it (and if they ask, we should inquire what they cannot do with the rare subset)
- these axioms should be added automatically based on the rare subset annotation

@cmungall @matentzn, please let us know if you have concerns or agree with this decision before we go forward with removing these axioms. Thanks!

**Comment by matentzn** [comment]
I have have nothing to say, but this is important enough to get a Chris opinion in a tech call.

**Comment by sabrinatoro** [comment]
From Tech call 2024-03-29:
Chris - Yes, agree with decision on the GitHub issue. However, the “rare” subset annotation may be outdated. When the annotation is orphanet subset, this is from the mapping work. There is an upcoming meeting with Nico,Sabrina and Orphanet on how to move forward.
Nico - there is one other item to check on with rare gci. There are three “not rare” diseases so a little bit of of QC is lost.

**Comment by matentzn** [comment]
Update:

@sabrinatoro needs to delete the classes "rare or common", "rare" and "not rare" from the ontology. Two files will be changed.

**Comment by matentzn** [comment]
Can we deal with this in a meeting? I am not sure we should obsolete rare/common at the moment, just remove their usage in Mondo!

**Comment by kanems** [comment]
@sabrinatoro and @matentzn What was the final decision on keeping or removing MONDO:0021135 (Rare or not rare)  and it's children under the "disease characteristic" branch in Mondo? 
These are still flagged as obsoletion candidates:

![Image](https://github.com/user-attachments/assets/3a092921-5f0e-40eb-a0c5-f4bb67d43f27)

**Comment by sabrinatoro** [comment]
@kanems I am 90% sure that our final decision was to keep these terms (ie not obsolete them), in case they are being used by other groups. 
However, we do not use "rare" in annotations in Mondo. 
We instead rely on the rare subset to indicate that a term is a rare disease (see [here](https://mondo.readthedocs.io/en/latest/editors-guide/rare-disease-subset/) for more details)
I will remove the obsoletion tag. 

If you use these terms, you can continue using them. However, if you need to know which Mondo terms are "rare", please use the rare subset.
